core/src/main/scala/org/apache/spark/sql/crossdata/catalog/persistent/ZookeeperCatalog.scala

```diff
@@ -149,9 +184,9 @@ class ZookeeperCatalog(sqlContext: SQLContext, override val catalystConf: Cataly
 
   override def isAvailable: Boolean = {
     //TODO this method must be changed when Stratio Commons provide a status connection of Zookeeper
-    val value=XDContext.catalogConfig.getString("zookeeper.connectionString")
-    val address=value.split(":")
-    Try(new Socket(address(0), address(1).toInt)).map( _ => true).getOrElse(false)
+    val value = XDContext.catalogConfig.getString("zookeeper.connectionString")
+    val address = value.split(":")
+    Try(new Socket(address(0), address(1).toInt)).map(_ => true).getOrElse(false)
```

Comment:
This is not an issue with this PR but I just noticed that the probing socket is not being closed as it would if the code were something like:

`Try(new Socket(address(0), address(1).toInt)).map(s => { s.close; true}).getOrElse(false)`

Note that despite `Socket` implements `AutoCloseable`, it would autoclose the resource when used through try-with-resources.